Crash on configuration change when License dialog is open

### Version

2.2.2

### Source

Google Play

### Vault encryption

Yes (with biometric unlock)

### Device

Samsung S23 Ultra

### Android version

Android 13

### ROM

_No response_

### Steps to reproduce

1. Open and unlock Aegis
2. From the overflow menu, tap About
3. From the About screen, tap License
4. Rotate screen

### What do you expect to happen?

License dialog rotates as expected

### What happens instead?

Crash. Appears to be from using private Fragment constructors for implementers of `SimpleWebViewDialog`, so this also happens for `ChangelogDialog`.


### Log

```
FATAL EXCEPTION: main
Process: com.beemdevelopment.aegis, PID: 12138
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.beemdevelopment.aegis/com.beemdevelopment.aegis.ui.AboutActivity}: androidx.startup.StartupException: Unable to instantiate fragment com.beem
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4169)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4325)
	at android.app.ActivityThread.handleRelaunchActivityInner(ActivityThread.java:6484)
	at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:6341)
	at android.app.servertransaction.ActivityRelaunchItem.execute(ActivityRelaunchItem.java:71)
	at android.app.servertransaction.ActivityTransactionItem.execute(ActivityTransactionItem.java:45)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2574)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:226)
	at android.os.Looper.loop(Looper.java:313)
	at android.app.ActivityThread.main(ActivityThread.java:8762)

                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:604)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)
Caused by: androidx.startup.StartupException: Unable to instantiate fragment com.beemdevelopment.aegis.ui.dialogs.LicenseDialog: could not find Fragment constructor
	at androidx.fragment.app.FragmentManager$3.instantiate(SourceFile:59)
	at androidx.fragment.app.FragmentStateManager.<init>(SourceFile:4)
	at androidx.fragment.app.FragmentManagerImpl.restoreSaveStateInternal(SourceFile:16)
	at androidx.fragment.app.FragmentManagerImpl.attachController$androidx$fragment$app$FragmentManager(SourceFile:166)
	at androidx.activity.ComponentActivity$$ExternalSyntheticLambda2.onContextAvailable(SourceFile:156)
	at androidx.activity.ComponentActivity.onCreate(SourceFile:33)
	at androidx.fragment.app.FragmentActivity.onCreate(SourceFile:1)
	at com.beemdevelopment.aegis.ui.AegisActivity.onCreate(SourceFile:137)
	at com.beemdevelopment.aegis.ui.AboutActivity.onCreate(SourceFile:17)
```

--- a/app/src/main/java/com/beemdevelopment/aegis/ui/dialogs/ChangelogDialog.java
+++ b/app/src/main/java/com/beemdevelopment/aegis/ui/dialogs/ChangelogDialog.java
@@ -5,7 +5,7 @@ import android.content.Context;
 import com.beemdevelopment.aegis.R;
 
 public class ChangelogDialog extends SimpleWebViewDialog {
-    private ChangelogDialog() {
+    public ChangelogDialog() {
         super(R.string.changelog);
     }
 

--- a/app/src/main/java/com/beemdevelopment/aegis/ui/dialogs/LicenseDialog.java
+++ b/app/src/main/java/com/beemdevelopment/aegis/ui/dialogs/LicenseDialog.java
@@ -5,7 +5,7 @@ import android.content.Context;
 import com.beemdevelopment.aegis.R;
 
 public class LicenseDialog extends SimpleWebViewDialog {
-    private LicenseDialog() {
+    public LicenseDialog() {
         super(R.string.license);
     }
 

